vent/api/actions.py

```diff
@@ -388,12 +392,29 @@ def restore(self, backup_file):
             backup = Template(backup_file)
             options = ['repo', 'branch', 'version', 'built', 'namespace',
```

Comment:
these should probably be pulled from `sections` of the template file that is being read, rather than hardcoded.